node.gyp

```diff
@@ -23,7 +23,7 @@
     'node_v8_options%': '',
     'node_enable_v8_vtunejit%': 'false',
     'node_core_target_name%': 'node',
-    'node_lib_target_name%': 'node_lib',
+    'node_lib_target_name%': 'libnode',
```

Comment:
I think this would make the product..liblibnode.so/liblibnode.dylib on other platforms? (looks a bit funny, but probably not a big deal)

Reply:
> I think this would make the product..liblibnode.so/liblibnode.dylib on other platforms?

This is the line I see on linuxONE for example:
```
ar crsT /data/iojs/build/workspace/node-test-commit-linuxone/nodes/rhel72-s390x/out/Release/obj.target/libnode.a ...
                                                                                                       ^^^^^^^^^
```